PIM/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,213 @@
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+> [Info] By default, a certain number of attributes has been predefined in the *PIM Basic Set* attribute set.
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/PIM/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Click the attribute you want to edit in the list of attributes.   
+  The *Edit Attribute* view is displayed. The *Data* tab is preselected.
+
+  ![Edit attribute data](../../Assets/Screenshots/PIM/Settings/Attributes/EditAttribute_Data.png "[Edit attribute data]")
+
+  > [Info] Be aware that you can only edit the attribute name, description, its status and configuration. All other fields and toggles are locked.
+
+2. Edit the desired data of the attribute in the corresponding fields in the *Data* tab.
+
+3. If desired, click the *Attribute Set Assignments* tab to check to which sets the selected attribute is assigned.
+
+  ![Edit attribute assignments](../../Assets/Screenshots/PIM/Settings/Attributes/EditAttribute_Assignments.png "[Edit attribute assignments]")
+
+   > [Info] If you want to change the attribute set assignments of the selected attribute, you have to modify the corresponding attribute set, see [Manage the attribute sets](./02_ManageAttributeSets.md).
+
+4. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Edit Attribute* view is closed.  
+
+
+
+## Deactivate an attribute
+
+It is recommended to deactivate an attribute instead of deleting it to prevent any problems because of existing dependencies. If you deactivate an attribute, it is no longer available for new attribute sets and it is hidden in existing products with this attribute.
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+> [Info] By default, a certain number of attributes has been predefined in the *PIM Basic Set* attribute set.
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/PIM/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Click the attribute you want to deactivate in the list of attributes.   
+  The *Edit Attribute* view is displayed. The *Data* tab is preselected.
+
+  ![Edit attribute data](../../Assets/Screenshots/PIM/Settings/Attributes/EditAttribute_Data.png "[Edit attribute data]")
+
+2. Disable the *Active* toggle in the *Data* tab.
+
+   > [Info] If you want to reactivate an attribute, enable the *Active* toggle.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The attribute has been deactivated. The *Edit Attribute* view is closed.
+
+4. Press **F5** to initialize the *Core1 Platform* and to apply the changes.   
+
```

Comment:
A little info for the user:
The values previously saved in deactivated attributes are not deleted, but the attribute is no longer displayed when looking at a pim Product. If the attribute is then activated again, the values will all be visible again.